doc/source/serve/tutorials/batch.md

```diff
@@ -52,24 +52,24 @@ class MyBackend:
 
 :::{note}
 By default, Ray Serve performs *opportunistic batching*. This means that as
-soon as the batch handler is called, the method will be executed without
-waiting for a full batch. If there are more queries available after this call
-finishes, a larger batch may be executed. This behavior can be tuned using the
+soon as it calls the batch handler, it executes the method without
```

Comment:
Serve doesn't call the batch handler. It gets called by the user's code.

```suggestion
soon as the batch handler is called, Serve executes the method without
```

Reply:
reverted